<ui version="4.0" >
 <class>RunDlg</class>
 <widget class="QWidget" name="RunDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Run</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QColumnView" name="cvCommands" >
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="wgSettings" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="KPushButton" name="pbNewCommand" >
        <property name="text" >
         <string>New Command</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KPushButton" name="pbImportActivities" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>In Entwicklung...</string>
        </property>
        <property name="text" >
         <string>Import Activities</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="KPushButton" name="pbEditCommand" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KPushButton" name="pbDeleteCommand" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
